src/TriggerBinding/SqlTableChangeMonitor.cs

```diff
@@ -955,7 +955,7 @@ private SqlCommand BuildUpdateTablesPostInvocation(SqlConnection connection, Sql
                 IF @unprocessed_changes = 0 AND @current_last_sync_version < {newLastSyncVersion}
                 BEGIN
                     UPDATE {GlobalStateTableName}
-                    SET LastSyncVersion = {newLastSyncVersion}
+                    SET LastSyncVersion = {newLastSyncVersion}, LastAccessTime = GETDATE()
```

Comment:
Should we be using GETDATE() or GETUTCDATE()?

Reply:
It shouldn't matter I think - it's all local to the server (so unless that changes timezones or something weird). 

That being said - using UTC seems reasonable to avoid that potential edge case. And that's a good general thing to default to for stored timestamps anyways.

Reply:
Defaulted to GETUTCDATE() to avoid potential edge cases.